pages/c/_cluster/apps/charts/index.vue

```diff
@@ -42,21 +84,64 @@ export default {
 
   data() {
     return {
-      allRepos:            null,
-      category:            null,
-      operatingSystem:     null,
-      searchQuery:         null,
-      showDeprecated:      null,
-      showHidden:          null,
+      allRepos:             null,
+      category:             null,
+      operatingSystem:      null,
+      searchQuery:          null,
+      showDeprecated:       null,
+      showHidden:           null,
+      sliders:         sliderData,
+      chartMode:            'featured',
+      chartOptions:    [
+        {
+          label:       'Browse',
+          value:       'browse',
+        },
+        {
+          label: 'Featured',
+          value: 'featured'
+        }
+      ]
     };
   },
 
   computed: {
+
     ...mapGetters(['currentCluster']),
     ...mapGetters({ allCharts: 'catalog/charts', loadingErrors: 'catalog/errors' }),
 
     hideRepos: mapPref(HIDE_REPOS),
 
+    viewOptions() {
+      const out = [];
+
+      if ( this.hasDetail ) {
+        out.push({
+          labelKey: 'resourceDetail.masthead.detail',
```

Comment:
Please define these option values in a constant outside the component as common practice around the code.

Reply:
This code is not part of this PR.